### ISSUE TITLE
Broken aria reference caused by double quotes in aria-labelledby attribute

### ISSUE THREAD

**Issue by amberhinds** [PopupMaker/Popup-Maker#1088]
## Describe the bug

In recent versions of Popup Maker, the aria-labelledby attribute is broken. It cannot reference the element's ID because there are double quotes. This is caused by the use of esc_attr() for $labelledby on this line of code:

https://github.com/PopupMaker/Popup-Maker/blob/7757a541a341a260f7ede126c40a956bb35d0e66/templates/popup.php#L12

esc_attr() does not allow double quotes to pass through it, so it replaces them with `&quot;`. The recommended fix is to not escape this variable since it's getting a generated ID not data from a user input field. Or possibly move esc_att() up to line 9 and into the quotes. 

## Site information
Popup Maker version: 1.20.3

WordPress version: 6.7.1

## Expected behavior
The aria-labelledby attribute will work.

## Current behavior

Example:

<img width="645" alt="Image of Popup Maker code with double quotes in the attribute" src="https://github.com/user-attachments/assets/b6a36a6d-128b-41e0-8228-6f7ae6291297" />

Code visible by viewing source:
`aria-labelledby=&quot;pum_popup_title_6549&quot;`

## Steps to reproduce
1. Go to a page with a popup on it.
2. Trigger the popup as needed.
3. Inspect the code for the popup.
4. View the page source and do a find on the page for `aria-labelledby`

## Errors
This triggers errors in Accessibility Checker. 😄

**Comment by amberhinds** [comment]
Just noticed this ticket, which is labeled as an enhancement rather than a bug and that is essentially the same thing: https://github.com/PopupMaker/Popup-Maker/issues/1078 Sorry for the duplicate...

**Comment by danieliser** [comment]
This was merged and released yesterday, sorry for the delay, thanks for the PR.
